docs/home/contribute/style-guide.md

```diff
@@ -144,24 +142,56 @@ The output is similar to this:
     NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
     nginx    1/1       Running   0          13s    10.200.0.4   worker0
 
+### Versioning Kubernetes examples
+
+Code examples and configuration examples that include version information should be consistent with the accompanying text. Identify the Kubernetes version in the Before you begin section.
```

Comment:
Referring to a topic heading is one place I think we should use bold. And we do use bold for **Before you begin** in the bulleted list below. So I think this should be, 
"the Kubernetes version in the **Before you begin** section."